cypress/integration/creatorOnboardingFlows/creatorSettings.spec.js

```diff
@@ -59,6 +59,37 @@ describe('Creator Settings Page', () => {
     cy.url().should('equal', baseUrl);
   });
 
+  it('should update the colors on the form when a new brand color is selected', () => {
+    const color = '#25544b';
+    const rgbColor = 'rgb(37, 84, 75)';
+
+    cy.findByLabelText(/^Brand color/)
+      .clear()
+      .type(color)
+      .trigger('change');
+
+    cy.findByRole('button', { name: 'Finish' }).should(
+      'have.css',
+      'background-color',
+      rgbColor,
+    );
+
+    cy.findAllByRole('radio', { name: /members only/i })
+      .check()
+      .should('have.css', 'background-color', rgbColor)
```

Comment:
As discussed yesterday, we typically haven't tested for color in E2E tests, but in this particular case, I think it makes sense.

Reply:
For some reason, I do not see this spec in Cypress 😟

Reply:
Do you have the latest `main`, @msarit? I just double-checked that I still see the spec file and verified that I do. 😕

Reply:
ok i will update my local version of Ridhwana's branch with `main`